handler/src/main/java/io/netty/handler/ssl/SniHandler.java

```diff
@@ -94,6 +96,20 @@ public SniHandler(AsyncMapping<? super String, ? extends SslContext> mapping) {
     }
 
     /**
+     * Sets if the {@link Channel} should be closed on a handshake failure, default is {@code true}.
+     */
+    public final void setCloseOnHandshakeFailure(boolean closeOnHandshakeFailure) {
+        this.closeOnHandshakeFailure = closeOnHandshakeFailure;
+    }
+
+    /**
+     * Gets if the {@link Channel} should be closed on a handshake failure, default is {@code true}.
+     */
+    public final boolean getCloseOnHandshakeFailure() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) A getX() method which returns a boolean should be named isX() [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3ABooleanGetMethodName)